### PR TITLE
Link to @jessitron's blog post in our documentation guides

### PR DESCRIPTION
**Pull request by vaidehijoshi**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
Adds a link to @jessitron's great blog post, [“Just” don’t](https://jessitron.com/2020/06/26/just-dont), which prompts us to avoid exclusive language.

## Related Tickets & Documents
N/A

## QA Instructions, Screenshots, Recordings

Screen Shot 2020-06-29 at 9 49 53 AM

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [x] docs.dev.to
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -59,7 +59,12 @@ The docs are a collection of [Markdown files][markdown] that also utilize
 For more information on how to use GitDocs read the [GitDocs
 guide][gitdocs_guide].
 
-# Regarding style
+# Regarding language & style
+
+We ask that you avoid trivializing terms when contributing to documentation.
+This includes words like "just", "simply", "easy", "obvious", and
```
Comment (jacobherrington) [inline docs/contributing.md:65]: I know this is annoying, but I think we usually keep punctuation like commas and periods inside of quotations. I'm sure there are differing opinions on this, but I usually just default to MLA:

`"just," "simply," "easy," "obvious," and "straightforward."`

Not a reason not to merge, but seems best to remain consistent! :)

File: docs/contributing.md [hunk 2]
```diff
@@ -59,7 +59,12 @@ The docs are a collection of [Markdown files][markdown] that also utilize
...
+"straightforward". You can learn more about why we want to avoid this kind of
+language in [this blog post](https://jessitron.com/2020/06/26/just-dont).
```
Comment (benhalpern) [inline docs/contributing.md:67]: There's actually a DEV post from 2017 which got a ton of reads and really helped kick off this conversation in the first place.

Maybe consider maybe using that one instead, or not if you feel like this one does a better job describing the situation? 🙂

File: docs/contributing.md (hunk 1 above)
Comment (vaidehijoshi) [reply docs/contributing.md:65]: @jacobherrington looks like this got merged by @nickytonline before i could make this change! sorry about that. can probably come back to it at a later point, looks like we have other instances of this like in `architecture.md`

File: docs/contributing.md (hunk 2 above)
Comment (vaidehijoshi) [reply docs/contributing.md:67]: @benhalpern no reason to not link to it except that i didn't know that it existed :) do you have a link handy? i can add it in here as well.

File: docs/contributing.md (hunk 1 above)
Comment (nickytonline) [reply docs/contributing.md:65]: Oops my bad..

Comment (jacobherrington) [reply docs/contributing.md:65]: lol no biggie, it's not important :)

File: docs/contributing.md (hunk 2 above)
Comment (benhalpern) [reply docs/contributing.md:67]: Oh whoops, I thought I linked to it in that comment @vaidehijoshi 

https://dev.to/marcuscreo/the-4-letter-word-word-that-makes-my-blood-boil

Looks like this got merged and I missed my chance. 😄

I didn't feel super strongly so I'll just say that fate chose.

**Pull request merged**